tests/nanocontracts/test_token_creation.py

```diff
@@ -34,13 +34,14 @@ def withdraw(self, ctx: Context) -> None:
     def create_token(
         self,
         ctx: Context,
+        salt: bytes,
         token_name: str,
         token_symbol: str,
         amount: int,
         mint_authority: bool,
         melt_authority: bool,
     ) -> None:
-        self.syscall.create_token(token_name, token_symbol, amount, mint_authority, melt_authority)
+        self.syscall.create_token(salt, token_name, token_symbol, amount, mint_authority, melt_authority)
```

Comment:
Add another test to confirm that a contract is allowed to create two tokens with the same token symbol but with different salts.